admin/templates/get-started.php

```diff
@@ -76,23 +76,17 @@ function check_is_responsive_addons_greater() {
 			}
 			?>
 			<?php do_action( 'responsive_addons_getting_started_settings_tab' ); ?>
-			<?php if( ! is_plugin_active( 'responsive-add-ons/responsive-add-ons.php' ) ): ?>
+			<?php if ( ! is_plugin_active( 'responsive-add-ons/responsive-add-ons.php' ) ) : ?>
 				<div class="responsive-theme-tab responsive-theme-templates-tab" data-tab="templates">
 					<p class="responsive-theme-tab-name">Starter&nbsp;Templates</p>
 				</div>
 			<?php endif; ?>
-			<div class="responsive-theme-tab responsive-theme-plugins-tab" data-tab="plugins">
-				<p class="responsive-theme-tab-name">Useful&nbsp;Plugins</p>
-			</div>
-			<?php
-			if ( ! class_exists( 'Responsive_Addons_Pro' ) ) {
-				?>
-				<div class="responsive-theme-tab responsive-theme-freevspro-tab" data-tab="freevspro">
-					<p class="responsive-theme-tab-name">Free&nbsp;vs&nbsp;Pro</p>
+				<div class="responsive-theme-tab responsive-theme-blocks-tab" data-tab="blocks">
+					<p class="responsive-theme-tab-name">Blocks</p>
```

Comment:
Use sanitization at both places

Reply:
Sanitization is done in the commit
https://github.com/cyberchimps/responsive/pull/399/commits/0f659e069e2de9f0aab3e90d29d25e8e813be610